sonar-scala-plugin/src/test/resources/checks/StringLiteralDuplicatedScala.scala

```diff
@@ -0,0 +1,61 @@
+class MyClass {
+  def foo(str: String) = {
+    str match {
+      case "Alice" =>
+        val message = "Hello dear" + "Alice" // Noncompliant
```

Comment:
Let's also test the message of the issue at least once
```suggestion
        val message = "Hello dear" + "Alice" // Noncompliant {{Define a constant instead of duplicating this literal "Hello dear" 3 times.}}
```